Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,33 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+function GryffindorHouse(obj){
```

Comment:
Just a small formatting issue on the function name -> camelCase is a general convention for function names 👍